src/functions.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] [REQUIRED] Great project @fatmahussein , however, I can see that you disable eslint on this file. Let's enable eslint so that it can check for your JS code standard. Let's manually resolve the error and request for another review.